docs/404.html

```diff
@@ -0,0 +1 @@
+<!DOCTYPE html><html><head><meta name="viewport" content="width=device-width"/><meta charSet="utf-8"/><title>404: This page could not be found</title><meta name="next-head-count" content="3"/><link rel="preload" href="/_next/static/css/956076c3cfc3cdd1f234.css" as="style"/><link rel="stylesheet" href="/_next/static/css/956076c3cfc3cdd1f234.css" data-n-g=""/><noscript data-n-css=""></noscript><link rel="preload" href="/_next/static/chunks/main-b5ed44aa77dc150295ca.js" as="script"/><link rel="preload" href="/_next/static/chunks/webpack-95c2b224bccf352ee870.js" as="script"/><link rel="preload" href="/_next/static/chunks/framework.046ad580301c0562bf6d.js" as="script"/><link rel="preload" href="/_next/static/chunks/bd2f88a7.a0df4e32a6979d8dc39f.js" as="script"/><link rel="preload" href="/_next/static/chunks/commons.286c5ad94e4334e87283.js" as="script"/><link rel="preload" href="/_next/static/chunks/pages/_app-ee18302c229a75f22a1a.js" as="script"/><link rel="preload" href="/_next/static/chunks/pages/_error-211314b6f798e8e84b55.js" as="script"/></head><body><div id="__next"><div style="color:#000;background:#fff;font-family:-apple-system, BlinkMacSystemFont, Roboto, &quot;Segoe UI&quot;, &quot;Fira Sans&quot;, Avenir, &quot;Helvetica Neue&quot;, &quot;Lucida Grande&quot;, sans-serif;height:100vh;text-align:center;display:flex;flex-direction:column;align-items:center;justify-content:center"><div><style>body { margin: 0 }</style><h1 style="display:inline-block;border-right:1px solid rgba(0, 0, 0,.3);margin:0;margin-right:20px;padding:10px 23px 10px 0;font-size:24px;font-weight:500;vertical-align:top">404</h1><div style="display:inline-block;text-align:left;line-height:49px;height:49px;vertical-align:middle"><h2 style="font-size:14px;font-weight:normal;line-height:inherit;margin:0;padding:0">This page could not be found<!-- -->.</h2></div></div></div></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"statusCode":404}},"page":"/_error","query":{},"buildId":"JHyUBz6cH3HkDjC0pMyGO","nextExport":true,"isFallback":false,"gip":true}</script><script nomodule="" src="/_next/static/chunks/polyfills-546d3edee431eb7e5a13.js"></script><script src="/_next/static/chunks/main-b5ed44aa77dc150295ca.js" async=""></script><script src="/_next/static/chunks/webpack-95c2b224bccf352ee870.js" async=""></script><script src="/_next/static/chunks/framework.046ad580301c0562bf6d.js" async=""></script><script src="/_next/static/chunks/bd2f88a7.a0df4e32a6979d8dc39f.js" async=""></script><script src="/_next/static/chunks/commons.286c5ad94e4334e87283.js" async=""></script><script src="/_next/static/chunks/pages/_app-ee18302c229a75f22a1a.js" async=""></script><script src="/_next/static/chunks/pages/_error-211314b6f798e8e84b55.js" async=""></script><script src="/_next/static/JHyUBz6cH3HkDjC0pMyGO/_buildManifest.js" async=""></script><script src="/_next/static/JHyUBz6cH3HkDjC0pMyGO/_ssgManifest.js" async=""></script></body></html>
```

Comment:
The `docs/` folder will get auto-generated upon `make deploy`, so let's just get rid of it for now

Reply:
Ok done